model/team.py

```diff
@@ -21,6 +21,24 @@ def __init__(self,
         self.team_leads: Set[str] = set()
         self.members: Set[str] = set()
 
+    def get_attachment(self):
+        """Return slack-formatted attachment (dictionary) for team."""
+        text_pairs = [
+            ('Github ID', self.github_team_id),
+            ('Github Team Name', self.github_team_name),
+            ('Display Name', self.display_name),
+            ('Platform', self.platform),
+            ('Team Leads', '\n'.join(self.team_leads)
+                           if self.team_leads != set() else 'n/a'),
```

Comment:
You can remove the ternary if statement(s) since line 36 will check to see if any of the fields are empty or not, and replace them with `n/a`.